mlir/lib/Dialect/Arith/IR/ArithOps.cpp

```diff
@@ -190,6 +190,14 @@ LogicalResult arith::ConstantOp::verify() {
     return emitOpError(
         "value must be an integer, float, or elements attribute");
   }
+
+  // Intializing scalable vectors with elements attribute is not supported
+  // unless it's a vector splot.
```

Comment:
`s/splot/splat`, also move comment to `emitOpError` message? It's more descriptive and kills two birds with one stone :)